CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Master
 
+##### Enhancements
+
+* Allow specifying multiple subspec pod dependencies inline in the Podfile,
+  via `pod 'Pod', subspecs: ['Subspec1', 'Subspec2']`.  
```

Comment:
I'd rather we use the hash rocket syntax in this CHANGELOG entry, we should stick to a consistent way of doing things in guides/examples etc as to not confuse our users who are not familiar with Ruby.